src/openforms/forms/statistics.py

```diff
@@ -34,7 +34,7 @@ def export_registration_statistics(
     )
 
     _start_date = make_aware(datetime.combine(start_date, time.min))
-    _end_date = make_aware(datetime.combine(end_date, time.max))
+    _end_date = make_aware(datetime.combine(end_date, time.min))
```

Comment:
Why not till the end of the day?

Reply:
Mostly because of how the export UI is set up - the start date is set as "first of the current month", with the idea that it's likely they'll export this every month. It was easiest to then calculate the end date as first-day-of-next-month, and since that would be included in the next export, the time ranges must work with an open interval rather than closed, otherwise you get duplicated statistics.

Come to think of it, it would probably be better to default the data ranges to the previous month and it might not necessarily be harder to just pick the last day of the month and work with inclusive ranges instead.